README.md

```diff
@@ -41,23 +62,23 @@ List of supported keywords:
 | `@maximum {number} [err_msg]`                                                                                              | `@maximum 42 Must be < 42` | `z.number().max(42, "Must be < 42")` |
 | `@minLength {number} [err_msg]`                                                                                            | `@minLength 42`            | `z.string().min(42)`                 |
 | `@maxLength {number} [err_msg]`                                                                                            | `@maxLength 42`            | `z.string().max(42)`                 |
-| `@format {FormatType} [err_msg]`                                                                                           | `@format email`            | `z.string().email()`                 |
+| `@format {FormatType} [err_msg]`                                                                                           | `@format email`            | `z.email()`                          |
 | `@pattern {regex}` <br><br> **Note**: Due to parsing ambiguities, `@pattern` does _not_ support generating error messages. | `@pattern ^hello`          | `z.string().regex(/^hello/)`         |
 
 By default, `FormatType` is defined as the following type (corresponding Zod validator in comment):
 
 ```ts
 type FormatType =
-  | "date-time" // z.string().datetime()
-  | "date" // z.string().date()
-  | "time" // z.string().time()
-  | "duration" // z.string().duration()
-  | "email" // z.string().email()
-  | "ip" // z.string().ip()
-  | "ipv4" // z.string().ip()
-  | "ipv6" // z.string().ip()
-  | "url" // z.string().url()
-  | "uuid"; // z.string().uuid()
+  | "date-time" // z.iso.datetime()
+  | "date" // z.iso.date()
+  | "time" // z.iso.time()
+  | "duration" // z.iso.duration()
```

Comment:
Note to myself: we need to handle this. `z.string().datetime()` & cie are deprecated in favor of `z.iso.datetime()` & cie.